<launch>
	<rosparam command="load" file="$(find kuam_state_machine)/config/param.yaml" />
	
	<!-- Do Not Input 0.0 -->
	<arg name="using_aruco" default="False" />

	<!-- rviz -->
	<arg name="rviz" default="false" />

	<!-- nodes -->
	<node pkg="kuam_state_machine" type="state_machine.py" name="state_machine" output="screen">
		<param name="using_aruco" value="$(arg using_aruco)" />
	</node>
	<node pkg="kuam_state_machine" type="setpoint_tf_node" name="setpoint_tf" output="screen" />
	<node pkg="kuam_state_machine" type="setpoint_visual_node" name="setpoint_visual" output="screen" />
	
</launch>